README.md

```diff
@@ -190,6 +190,20 @@ if none of the requested sequences failed to download, the corresponding artifac
 
 If some run IDs failed to download they are returned in the `--o-failed-runs` artifact, which can be directly inputted as `--i-accession-ids` to a subsequent `get-sequence` command.
 
+#### Special case: Fetching restricted access sequences with a dbGAP repository key
+To get access to the respective dbGaP repository key users must first apply for approval and then retrieve the key from dbGAP (see prerequisites described [here](https://www.ncbi.nlm.nih.gov/sra/docs/sra-dbgap-download/)).    
+
+To retrieve sequencing data using the acquired dbGAP repository key, without revealing the sensitive key, set the filepath to the stored key as an environment variable. You can either do this by running the following command in your terminal `export KEY_FILEPATH=<path to key>` or by adding the variable assignment to your `.env` file. For the latter option, make sure to ignore this file in version control (add to `.gitignore`).        
+Having set the filepath of the key as an environment variable you can fetch the sequencing data by running `get-sequences` with the parameter `--p-restricted-access`:
+```shell
+qiime fondue get-sequences \
+              --i-accession-ids ids.qza \
+              --p-email your_email@somewhere.com \
+              --p-restricted-access \
+              --output-dir output_path
+```
+__Note:__ Fetching metadata with a dbGAP repository key is not supported. Hence, this flag is only available in the `get-sequences` action (and not in the `get-metadata` and `get-all` actions).
```

Comment:
Is this because it's not possible in the first place? Or are you planning on adding support for this later?

Reply:
As far as I understand it, the metadata is provided "manually" upon getting access to the restricted access data. I am planning to look into this more in detail but as far as my understanding goes for now this is not possible.